pages/apis/rest_api/pipelines.md

```diff
@@ -415,6 +415,17 @@ Error responses:
 </tbody>
 </table>
 
+### On slug creation conventions
+
+Pipeline slugs are generated based on the pipeline name you provide during the pipeline creation.
+The maximum allowed character length for a pipeline slug is `100`.
+The supported character format for slug generation is:
+`/\A[a-zA-Z0-9]+[a-zA-Z0-9\-]*\z/`.
+
+All the whitespace characters that appear in the pipeline name consecutively will be converted to a single `-` character (so `"Hello[space]there"` and `"Hello[space, space, space, etc.]there"` will be equally converted to a `hello-there` slug), and uppercase will be converted to lowercase.
+
+An attempt at creating a new pipeline with a name that matches an existing pipeline name will throw an error.
+
```

Comment:
For all the changes to make to `pages/apis/graphql/cookbooks/pipelines.md` above, make/reflect them in this file too.

Reply:
Done.